sys/net/include/aodvv2/aodvv2.h

```diff
@@ -34,7 +35,7 @@ extern "C" {
 /**
  * @brief   Initialize the AODVv2 routing protocol.
  */
-void aodv_init(void);
+void aodv_init(kernel_pid_t interface);
```

Comment:
Documentation of interface. What is it for? And what's with multiple interface support?